spec/stringProcessing/relevantWordsDutchSpec.js

```diff
@@ -0,0 +1,40 @@
+let WordCombination = require( "../../js/values/WordCombination" );
+let relevantWords = require( "../../js/stringProcessing/relevantWords" );
+let getRelevantWords = relevantWords.getRelevantWords;
+let dutchFunctionWords = require( "../../js/researches/dutch/functionWords.js" )().all;
+
+describe( "gets Dutch Word Combinations", function() {
```

Comment:
Is there a specific reason that `Word` and `Combinations` are capitalized? Otherwise I'd suggest to change this.